phpstan.neon

```diff
@@ -1,17 +1,9 @@
-includes:
-    - vendor/szepeviktor/phpstan-wordpress/extension.neon
 parameters:
-    #level: max
-    level: 5
+    level: 5 # Increase until "max"
     paths:
-        - lib/
-    scanFiles:
-        # Plugin stubs
-        - vendor/php-stubs/wp-cli-stubs/wp-cli-stubs.php
```

Comment:
ahhh, here's where that `wp-cli-stubs` was being used. I notice that this is heavily thinned out. What are we losing without the `phpstan-wordpress/extension.neon`?

also, in terms of `"le mieux est l'ennemi du bien` — should implement level `1` and then in separate PRs work our way up?

Reply:
🤯 Removing `phpstan-wordpress/extension.neon`? 💣 It is like pure gold!

`phpstan/extension-installer` package includes it automatically. 🤖